src/Plugin.php

```diff
@@ -211,6 +218,13 @@ public static function init() {
       }
     }
 
+    // Schedules delete orphan variants cron.
```

Comment:
Apparently we are putting our crons here. 
Click the arrow to see other crons added in the past.